src/mlpack/core/util/io.hpp

```diff
@@ -34,12 +35,10 @@
 namespace mlpack {
 namespace util {
 
-// Externally defined in option.hpp, this class holds information about the
-// program being run.
-class ProgramDoc;
+// This class holds information about the program being run.
+struct BindingDetials;
```

Comment:
```suggestion
struct BindingDetails;
```
Oops, looks like a typo. :)  Although, if the build passed without the right name here, maybe it is an unnecessary forward declaration and we can remove it?

Reply:
Removed. :)